### PR TITLE
Update RCE toolbar a11y

### PR DESCRIPTION
**Pull request by vanwagonet**
extra: make it look like the new one

refs: MBL-12434, MBL-12435
affects: Student, Teacher
release note: Improved accessibility of rich text editor

### CODE REVIEW

File: Canvas/Canvas/CanvasAppDelegate.swift [hunk 1]
```diff
@@ -60,11 +60,12 @@ class AppDelegate: UIResponder, UIApplicationDelegate, AppEnvironmentDelegate {
 
         if let session = Keychain.mostRecentSession {
             window?.rootViewController = LoadingViewController.create()
+            window?.makeKeyAndVisible()
             userDidLogin(keychainEntry: session)
         } else {
             window?.rootViewController = LoginNavigationController.create(loginDelegate: self, fromLaunch: true)
+            window?.makeKeyAndVisible()
         }
-        window?.makeKeyAndVisible()
```
Comment (vanwagonet) [inline Canvas/Canvas/CanvasAppDelegate.swift]: this was causing the CanvasCore.LocalizationManager to not be able to display the restart alert when the device switched locales with a logged in user with a different locale

File: Core/Core/RichContentEditor/RichContentToolbarView.xib [hunk 2]
```diff
@@ -76,7 +76,7 @@
                                                 <constraint firstAttribute="height" constant="44" id="BJX-9x-KpX"/>
                                                 <constraint firstAttribute="width" constant="44" id="NVn-un-JZf"/>
                                             </constraints>
-                                            <color key="tintColor" red="0.1764705882352941" green="0.23137254901960785" blue="0.27058823529411763" alpha="1" colorSpace="calibratedRGB"/>
+                                            <color key="tintColor" red="0.17647058823529413" green="0.23137254901960785" blue="0.27058823529411763" alpha="1" colorSpace="custom" customColorSpace="sRGB"/>
```
Comment (vanwagonet) [inline Core/Core/RichContentEditor/RichContentToolbarView.xib]: the color was a little off in the hex, which caused the name to not get picked up right.

File: rn/Teacher/src/common/components/rich-text-editor/__tests__/RichTextToolbar.test.js [hunk 3]
```diff
@@ -51,20 +49,6 @@ describe('RichTextToolbar', () => {
...
-        <RichTextToolbar
-          {...defaultProps}
-          redo={jest.fn()}
-          undo={jest.fn()}
-        />
-      ).toJSON()
-    ).toMatchSnapshot()
-    I18nManager.isRTL = false
-  })
-
```
Comment (vanwagonet) [inline rn/Teacher/src/common/components/rich-text-editor/__tests__/RichTextToolbar.test.js]: the `icon()` function uses bundled images with directional metadata, so they will get displayed properly, and don't need to be manually flipped.

File: rn/Teacher/src/images/index.js [hunk 4]
```diff
@@ -52,22 +52,6 @@ const images = {
...
-    unorderedList: require('./rce/unordered-list.png'),
-    redo: require('./rce/redo.png'),
-    undo: require('./rce/undo.png'),
-    active: {
-      bold: require('./rce/bold-active.png'),
-      italic: require('./rce/italic-active.png'),
-      orderedList: require('./rce/ordered-list-active.png'),
-      unorderedList: require('./rce/unordered-list-active.png'),
-    },
-  },
```
Comment (vanwagonet) [inline rn/Teacher/src/images/index.js]: look at all the duplicate icons that can go bye bye!

**Comment by inst-danger** [comment]
#### Release Note: 
Improved accessibility of rich text editor

#### Affected Apps: Student, Teacher

[MBL-12434](https://instructure.atlassian.net/browse/MBL-12434)
[MBL-12435](https://instructure.atlassian.net/browse/MBL-12435)

Coverage | New % | Master % | Delta
-------- | ----- | -------- | -----
**Core** | 95.16% | 95.16% | 0%
**Student** | 92.84% | 92.84% | 0%
**Teacher** | 100% | 100% | 0%
**React Native** | 92.68% | 92.71% | -0.03%
<p align="right">
  Generated by :no_entry_sign: <a href="http://github.com/danger/danger-js/">dangerJS</a>
</p>

**Pull request merged**